workflow_VHcc.py

```diff
@@ -144,8 +144,8 @@ def count_objects(self, variation):
 
 
     def evaluateBDT(self, data):
-        print(self.params.Models.BDT[self.channel][self.events.metadata["year"]].model_file)
-        print()
+        #print(self.params.Models.BDT[self.channel][self.events.metadata["year"]].model_file)
```

Comment:
In principle, this can be deleted, not just commented out, if you see it work

Reply:
It can stay, as it may be useful for debugging later